cmd/influxd/server_integration_test.go

```diff
@@ -66,6 +68,26 @@ type Node struct {
 // Cluster represents a multi-node cluster.
 type Cluster []*Node
```

Comment:
Can you change the `Cluster` to be:

```go
type Cluster struct {
    Nodes []*Node
}
```

The `range *c` is confusing below and I think it would make more sense to have `range c.Nodes`.